src/NuGet.Core/NuGet.Commands/RestoreCommand/ProjectRestoreCommand.cs

```diff
@@ -167,7 +183,44 @@ public ProjectRestoreCommand(ProjectRestoreRequest request)
             return Tuple.Create(success, graphs, allRuntimes);
         }
 
-        private async Task<DownloadDependencyResolutionResult> ResolveDownloadDependencies(RemoteWalkContext context, ConcurrentDictionary<LibraryRange, Task<Tuple<LibraryRange, RemoteMatch>>> downloadDependenciesCache, ProjectModel.TargetFrameworkInformation targetFrameworkInformation, CancellationToken token)
+        /// Gets the runtime graph specified in the path.
+        /// returns null if an error is hit. A valid runtime graph otherwise.
```

Comment:
missing the xml part of xmldoc

Reply:
This is private, so didn't mean for it to be an xml comment.